build.gradle

```diff
@@ -0,0 +1,84 @@
+plugins {
+    id 'java'
+    id 'org.jetbrains.kotlin.jvm' version '1.4.21'
+    id 'com.palantir.docker' version '0.25.0'
+    id 'application'
+}
+
+group 'com.exactpro.th2'
+version release_version
+
+sourceCompatibility = JavaVersion.VERSION_11
+targetCompatibility = JavaVersion.VERSION_11
+
+compileKotlin {
+    kotlinOptions {
+        jvmTarget = "11"
+
+    }
+}
+
+compileTestKotlin {
+    kotlinOptions {
+        jvmTarget = "11"
+    }
+}
+
+dependencies {
+    // BOM with external dependencies restrictions
+    api platform('com.exactpro.th2:bom:3.0.0')
```

Comment:
Why not 4.*?

Reply:
It is nearest for origin bom used in gitlab